### DISCUSSION TITLE
Chrome cast for watching anime on the TV

### DISCUSSION THREAD

**Discussion by ThatTakashi** [aniyomiorg/aniyomi#221] [Ideas]
Would it at all be possible to support chrome cast in the aniyomi app?

**Comment by Aman9das** [comment]
Yes, it is already possible, using external video players :)

More > Settings > player > external player preference.

VLC is great if you are wondering

**Reply by Avellea** [reply]
That's not a solution, that's a workaround. The best way to do this would be to support Chromecast natively in the embedded video player.

**Reply by Aman9das** [reply]
Ok this is the thing, everything can be called a workaround...

I may ask why doesnt my android device not come with a torrent client pre-installed.. Why does it not provide an rss reader out-of-the-box?

But most people don't use these services, unfortunately. And it'll be nothing but feature creep for them. Every software need not do everything, VLC specializes in handling media and Aniyomi should trust VLC to do it's thing.

**Reply by Avellea** [reply]
> Ok this is the thing, everything can be called a workaround...
> 
> I may ask why doesnt my android device not come with a torrent client pre-installed.. Why does it not provide an rss reader out-of-the-box?
> 
> But most people don't use these services, unfortunately. And it'll be nothing but feature creep for them. Every software need not do everything, VLC specializes in handling media and Aniyomi should trust VLC to do it's thing.

Normally I'd agree with you, but the Gogoanime extension doesn't support opening in VLC. VLC claims it can't open the link, and I haven't found a single subbed or dubbed anime on zoro.to extension. 

I can't say for the other extensions, but given that Gogoanime is probably the most popular option, and Zoro.to is only available to the Japanese audience, that's leaving a lot of users in the dark until Gogo and/or VLC get a fix, or Aniyomi implements native Chromecast support.

**Reply by Avellea** [reply]
No edit button on mobile, but I'd like to point out  issue #78. We should move the conversation there if it is to continue.

**Reply by joelchrono12** [reply]
I confirm I cannot open gogoanime content with VLC 
But I also agree that media chromecasting should be handled by proper video players

**Reply by Aman9das** [reply]
Haha you use aniyomi too, Joel?

Anyway this seems to an issue in the gogoanime extension, its sending the link to external video player with an expiry token included.

It used to work 2 months ago certainly. A web based project like this needs constant maintainance.. Not possible for a person alone, what jmir1 has achieved is tremendous! If only i  could help, maybe i will learn kotlin one day..
